freeze.py

```diff
@@ -292,8 +292,8 @@ def find_files(directory, patterns):
     # for certain distros (like Fedora, openSUSE, Debian, etc...)
     # Also add Glib related files (required for some distros)
 
-    for added_lib in [ARCHLIB + "libssl.so.1.0.0",
```

Comment:
Apparently these files no longer have versions appended to them on my Linux build server. Hmmm. Also, I can't find `ARCHLIB + "libthread-2.0.so"`. Is that supposed to be `libpthread.so`?

Reply:
@jonoomph 

I think it's supposed to be `libpthread-2.30.so`, actually, AKA `libpthread.s.0`. That exists on my Fedora machine. Or there's `libthread_db.so.1` `libthread_db-1.0.so`, also part of glibc. But it's most likely meant to be the former.

The unversioned `libname.so` files are usually installed by `-dev` packages, they're for use when linking builds against the libraries. If you don't have the versioned `.so.M.N...` files, though, it's likely you either have devel packages installed without the corresponding libs, or you really need to run `ldconfig`. :laughing: